### ISSUE TITLE
Add a progress saved message to autosaves

### ISSUE THREAD

**Issue by Konclan** [TeamSpen210/BEE2-items#983]
I feel like sometimes people get scared or confused by auto-saves when they didn't know the map had them and the fear of losing progress if they don't use F6 to quicksave so maybe adding a message that appears on screen when a player hits an auto-save will help them be reassured that they can die and not lose all progress.

**Comment by TeamSpen210** [comment]
Note to self: `PORTAL2_AutoSave` localises to 'AUTO SAVE'. That should work.

**Comment by Byzarru** [comment]
This should be an option.

**Comment by LautaroL20** [comment]
I agree with @Byzarru, this must be an option inside stylevar. I really don´t want to display "magical letters" in the screen...

**Comment by TheDarkBomber** [comment]
How about just showing the quick save display as the auto save display?

**Comment by cartman1337** [comment]
Since there are two auto save functions in Beemod, and not everybody wants to display text that progress has been saved, how about adding text to the Save Point variant, and leave the Trigger Autosave variant without, or vice versa?

**Comment by LautaroL20** [comment]
Merging that inside the Save Point wouldn´t be that good. Save Point was created with the sole purpose to saving the player progress once this reaches the markered area (Not really since it acts the exactly same  as Trigger autosave due to a bug ).

So, the conclution is that this must be a separated item.

**Comment by cartman1337** [comment]
I don't get it. Isn't the point of Trigger Autosave to save the player progress once she reaches the marked area too? What is the intended difference between the two?

**Comment by LautaroL20** [comment]
Trigger autosave is supposed to cover the entirely block, and Save Point would cover the circular area only, simple as that.

**Comment by TeamSpen210** [comment]
The SavePoint is intended to save when you reach the marked voxel - the cube it's in. That's not a bug.
